src/planner/update_plan.cpp

```diff
@@ -31,6 +31,7 @@ UpdatePlan::UpdatePlan(storage::DataTable *table,
       auto col_id = target.first;
       update_primary_key_ =
           target_table_->GetSchema()->GetColumn(col_id).IsPrimary();
+      std::cout << "col_id = " << col_id << " " << update_primary_key_ << std::endl;
```

Comment:
@yingjunwu The source validator complains, that `std::cout` is not allowed. [1] Is this a debug line?

[1] https://travis-ci.org/cmu-db/peloton/jobs/362378845#L5757

Reply:
@tcm-marcel yes nice catch! I've fixed it. Thanks!